src/bfd-disasm.h

```diff
@@ -1,19 +1,27 @@
 #pragma once
 
 #include "disasm.h"
+#if __has_include(<filesystem>)
+#include <filesystem>
+namespace std_filesystem = std::filesystem;
+#elif __has_include(<experimental/filesystem>)
+#include <experimental/filesystem>
+namespace std_filesystem = std::experimental::filesystem;
+#else
+#error "neither <filesystem> nor <experimental/filesystem> are present"
+#endif
```

Comment:
Can this be extracted out into another file, maybe `filesystem_wrapper.h`? I see another copy paste of this in `utils.cpp`. Would be great to not repeat this and also avoid using transitively included code.

Reply:
@danobi sure